### PR TITLE
(vi)nettruyen.com Fix encrypted pages

### PR DESCRIPTION
**Pull request by hajaulee**

### CODE REVIEW

File: src/vi/nettruyen/src/eu/kanade/tachiyomi/extension/vi/nettruyen/NetTruyen.kt [hunk 1]
```diff
@@ -152,12 +157,51 @@ class NetTruyen : ParsedHttpSource() {
...
+        val keyIndex = scriptTag.indexOf("var k=") + 5
+        val listIndex = scriptTag.indexOf("var l=") + 5
+        val key = scriptTag.substring(keyIndex + 2, scriptTag.indexOf('"', keyIndex+2))
+        val list = scriptTag.substring(listIndex + 1, scriptTag.indexOf(";", listIndex))
+        val jsonList = JSONArray(list)
+
+        for (i in 0 until jsonList.length()) {
+            val imageUrl = decrypt(jsonList.getString(i), key)
+            val imageUri = if (imageUrl!!.startsWith("//")) "http:$imageUrl" else imageUrl
+            Log.d("Page:", imageUri)
```
Comment (arkon) [inline src/vi/nettruyen/src/eu/kanade/tachiyomi/extension/vi/nettruyen/NetTruyen.kt]: Remove this please

Comment (hajaulee) [reply src/vi/nettruyen/src/eu/kanade/tachiyomi/extension/vi/nettruyen/NetTruyen.kt]: Remove `Log.d("Page:", imageUri)`?

Comment (arkon) [reply src/vi/nettruyen/src/eu/kanade/tachiyomi/extension/vi/nettruyen/NetTruyen.kt]: Yes, the debug log.

Comment (hajaulee) [reply src/vi/nettruyen/src/eu/kanade/tachiyomi/extension/vi/nettruyen/NetTruyen.kt]: Ready

**Comment by hajaulee** [comment]
Ready

**Pull request merged**